File: server-scala-play-server/app/model/LegResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for LegResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class LegResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  appKey: Option[String],
  distance: Option[Double],
  duration: Option[Long],
  startLatitude: Option[Double],
  startLongitude: Option[Double],
  startAltitude: Option[Double],
  startDate: Option[Long],
  endLatitude: Option[Double],
  endLongitude: Option[Double],
  endAltitude: Option[Double],
  endDate: Option[Long],
  created: Option[Long],
  tags: Option[String],
  deviceId: Option[String],
  accountId: Option[Long],
  deviceSignature: Option[String],
  parentAccountId: Option[Long],
  steps: Option[List[StepResponse]],
  `type`: Option[String],
  returning: Option[String]
)

object LegResponse {
  implicit lazy val legResponseJsonFormat: Format[LegResponse] = Json.format[LegResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/LegResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class LegResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public string AppKey { get; set; }
    public double Distance { get; set; }
    public long Duration { get; set; }
    public double StartLatitude { get; set; }
    public double StartLongitude { get; set; }
    public double StartAltitude { get; set; }
    public long StartDate { get; set; }
    public double EndLatitude { get; set; }
    public double EndLongitude { get; set; }
    public double EndAltitude { get; set; }
    public long EndDate { get; set; }
    public long Created { get; set; }
    public string Tags { get; set; }
    public string DeviceId { get; set; }
    public long AccountId { get; set; }
    public string DeviceSignature { get; set; }
    public long ParentAccountId { get; set; }
    public List<StepResponse> Steps { get; set; }
    public string Type { get; set; }
    public string Returning { get; set; }
}


